core/src/main/java/pl/agh/edu/enums/Frequency.java

```diff
@@ -0,0 +1,23 @@
+package pl.agh.edu.enums;
+
+import pl.agh.edu.model.time.Time;
+
+import java.time.LocalDateTime;
+
+public enum Frequency{
+    EVERY_SHIFT,
+    EVERY_UPDATE,
+    EVERY_DAY,
+    EVERY_MONTH,
+    EVERY_YEAR;
+
+    public LocalDateTime add(LocalDateTime localDateTime){
+        return switch (this){
+            case EVERY_SHIFT -> localDateTime.plusHours(8);
+            case EVERY_UPDATE -> localDateTime.plusMinutes(Time.timeUnitInMinutes);
```

Comment:
może by nazwę zmienić na EVERY_TICK albo EVERY_TIME_TICK? Choć jak już ogarnąłem o co z tym chodzi to już takie dziwne się nie wydaje.

Reply:
zrobione na EVERY_TIME_TICK